### ISSUE TITLE
Sumstats not in standard format

### ISSUE THREAD

**Issue by ljwh2** [EBISPOT/goci#1056]
James confirmed Kettunen (PMID 27005778) and Draisma (26068415) will not enter the new harmonisation pipeline since only the raw files are available and not any formatted files (even though these were formatted before). These studies need to be formatted to gwas-ssf standard so they can be harmonised.  

For Suhre, the raw files have no header. I couldn't replicate the issue described in gwas-curator-tasks#17. For the files I checked (including GCST90101376), the p value was there in both raw and harmonised files.

**Comment by ljwh2** [comment]
see also https://app.zenhub.com/workspaces/gwas-59df823c4a6feb3786810391/issues/gh/ebispot/xgwas-curator-tasks/17

**Comment by earlEBI** [comment]
@jdhayhurst Draisma has no chromosome and bp info, only has rsID - is it still worth formatting these with NA in those columns?

**Comment by earlEBI** [comment]
@jdhayhurst Do you need the following hidden or deleted?

Current raw files (named 'GCSTxxx_build.txt.gz') on staging
Harmonised folders on public
Current yaml files

**Comment by ljwh2** [comment]
@earlEBI  added comment about Suhre in original text

**Comment by earlEBI** [comment]
@jdhayhurst  Will this be sufficient to allow harmonising?

**Comment by earlEBI** [comment]
For the record, if it is decided these files do need to be re-harmonised, further info:
Kettunen: All required data is present. I could add formatted files to SSF standard. 
Draisma: Only chr and bp is missing, but rs_id is present. I could add formatted files meeting old standard. 
Suhre: Other allele and EAF is missing. Will not go into pipeline. However, I could add the missing header and format to old standard.

**Comment by ljwh2** [comment]
Discussed Suhre on Slack and agreed:
1. @earlEBI to fix the raw files so they have the correct headers
2. @jdhayhurst to create yaml files for these and the existing harmonised files. Files will not be re-harmonised.

**Comment by earlEBI** [comment]
1. Kettunen - I've added the formatted files and they've been released to pub ftp (and kept the raw files under .txt.gz)
2. Draisma - I've added the formatted files and they've been released to pub ftp (and removed the old .bz2 files)
      - except GCST90240007 which should release overnight. 
4. Suhre - I've added the header to the raw files, released to pub ftp (replaced old files)

I think yaml files need to be updated (md5sum) for Suhre and Kettunen
